lib/skate_web/channels/notifications_channel.ex

```diff
@@ -0,0 +1,23 @@
+defmodule SkateWeb.NotificationsChannel do
+  use SkateWeb, :channel
+
+  @impl Phoenix.Channel
+  def handle_info({:notification, message}, socket) do
+    valid_token? =
+      Application.get_env(:skate, :valid_token?, &SkateWeb.ChannelAuth.valid_token?/1)
+
+    if valid_token?.(socket) do
+      :ok = push(socket, "notification", %{data: message})
+      {:noreply, socket}
+    else
+      :ok = push(socket, "auth_expired", %{})
+      {:stop, :normal, socket}
+    end
+  end
+
+  @impl true
+  def join("notifications", _message, socket) do
```

Comment:
This doesn't have a way to specify which routes to subscribe to. If the intent is to push that off to keep this PR smaller, let's add another ticket ahead of [Create front end socket](https://app.asana.com/0/1148853526253426/1187690663845805), so that ticket can stay focused on the frontend half.

Reply:
@skyqrose see my comment on the PR above--essentially, while you were out, we clarified part of the UX related to waivers that affect multiple routes, and as a consequence I decided it would make more sense to have all notifications go through a common channel, such that all clients get all notifications, and it's up to the frontend to decide whether to display a notification or not.

Reply:
Ah, I missed that comment. 👍